src/asmcnc/skavaUI/widget_virtual_bed.py

```diff
@@ -110,13 +113,36 @@ class VirtualBed(Widget):
     # G54: workpiece co-ordinates
     # G28: set reference point
 
+
+    width_modifier = NumericProperty()
+    x_pos_modifier = NumericProperty()
+
     def __init__(self, **kwargs):
     
         super(VirtualBed, self).__init__(**kwargs)
         self.m=kwargs['machine']
         self.sm=kwargs['screen_manager']
+        self.set_up_virtual_bed(0)
+
+    def set_up_virtual_bed(self, dt):

```

Comment:
When we define functions that are called by clock functions, I think it would make sense to put dt=0 as the parameter so that we don't have to pass in the 0 when we manually call the function

Reply:
Have made the change and tested on the rig :)